audbackend/core/backend.py

```diff
@@ -321,28 +321,73 @@ def latest_version(
 
     def _ls(
             self,
-            path: str,
-    ) -> typing.List:  # pragma: no cover
-        r"""List content of path."""
+            folder: str,
+    ) -> typing.List[typing.Tuple[str, str, str]]:  # pragma: no cover
+        r"""List all files under folder.
+
+        Returns an empty list if folder does not exist.
+
+        """
         raise NotImplementedError()
```

Comment:
We need to add documentation that a developer of a new backend should return an empty list for empty folders (if actual folders exist on the backend). Should we add a comment here or should we add a special developer section to the documentation?

Reply:
I changed the comment to: `Return an empty list if no files match or folder does not exist.` 

And yes, I plan to include a usage section where we discuss how to implement a custom backend.

Reply:
Good.

I also created https://github.com/audeering/audbackend/issues/60 to track the documentation part.